src/com/lftechnology/java/training/niraj/ems/services/implementations/UserServiceImpl.java

```diff
@@ -0,0 +1,507 @@
+        boolean isDeleted = false;
+        if (employee != null) {
+            employee.setStatus(Status.INACTIVE);
+            isDeleted = employeeDao.update(employee);
+        } else {
+            throw new CustomException(Constants.EMPLOYEE_NOT_FOUND);
+        }
+        return isDeleted;
+    }
+
+    /**
+     * Gets user registration informations
+     * 
+     * @author Niraj Rajbhandari<nirajrajbhandari@lftechnology.com>
+     * 
+     * @param scanner
+     * @param console
+     * @return
+     */
+    public Map<String, String> getUserRegistrationInfo(Scanner scanner, Console console) {
+        Map<String, String> userRegistrationInfo = new LinkedHashMap<String, String>();
+        userRegistrationInfo.put(Constants.FULLNAME, userInput.getInput(scanner, console, Constants.FULLNAME_LABEL));
+        userRegistrationInfo.put(Constants.USERNAME, userInput.getInput(scanner, console, Constants.USERNAME_LABEL));
+        userRegistrationInfo.put(Constants.PASSWORD, userInput.getInput(scanner, console, Constants.PASSWORD_LABEL));
+        userRegistrationInfo.put(Constants.ADDRESS, userInput.getInput(scanner, console, Constants.ADDRESS_LABEL));
+        LOGGER.info(Constants.DEPARTMENT_LABEL);
+        int department = userNumInput.getInput(scanner, 1, 8);
+        userRegistrationInfo.put(Constants.DEPARTMENT, getDepartmentById(department));
+        LOGGER.info(Constants.ROLE_LABEL);
+        int role = userNumInput.getInput(scanner, 1, 2);
+        if (role == 1) {
+            userRegistrationInfo.put(Constants.ROLE, Roles.ADMIN.getRole());
+        } else {
+            userRegistrationInfo.put(Constants.ROLE, Roles.USER.getRole());
+        }
+
+        return userRegistrationInfo;
+    }
+
+    /**
+     * Gets user credentials
+     * 
+     * @author Niraj Rajbhandari <nirajrajbhandari@lftechnology.com>
+     * @param scanner
+     * @param console
+     * @return
+     */
+    public Map<String, String> getUserCredentials(Scanner scanner, Console console) {
+        Map<String, String> credentials = new LinkedHashMap<String, String>();
+        credentials.put(Constants.USERNAME, userInput.getInput(scanner, console, Constants.USERNAME_LABEL));
+        credentials.put(Constants.PASSWORD, userInput.getInput(scanner, console, Constants.PASSWORD_LABEL));
+        return credentials;
+    }
+
+    /**
+     * Registers a user
+     * 
+     * @author Niraj Rajbhandari <nirajrajbhandari@lftechnology.com>
+     * @param employeeInfo
+     * @return
+     * @throws SQLException
+     */
+    public Map<String, String> registerUser(Map<String, String> employeeInfo) throws SQLException {
+        Map<String, String> userExistsCondition = new LinkedHashMap<String, String>();
+        Map<String, String> employeeAfterRegistration = null;
+        userExistsCondition.put(Constants.USERNAME, employeeInfo.get(Constants.USERNAME));
+        System.out.println(employeeDao.exists(userExistsCondition));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)